nss-ice-phantoms/BangazonSite
# Completed ticket number 3

# Description
completed Ticket #3 

## Type of change

# Testing Instructions

1. `git fetch --all`
2. `git checkout AP-Create`
3. Run BangazonSite
4. Click "Sell", check to see if the list of products are specific to the user
5. try to create a new product to check if the new product belongs to the user 
6. Make sure after creating an new a product that the site routes to the details page of the newly created product


# Checklist:
